lib/src/rules/discarded_futures.dart

```diff
@@ -44,13 +44,22 @@ Future<void> createDir(String path) async {}
 ''';
 
 class DiscardedFutures extends LintRule {
+  static const LintCode code = LintCode('discarded_futures',
+      "Asynchronous function invoked in a non-'async' function.",
+      correctionMessage:
+          "Try converting the enclosing function to be 'async' and then "
+          "'await' the future");
```

Comment:
Should Future be capitalized?

Reply:
In other docs I've written (not sure whether there's a standard), I've capitalized "Future" when referring to the class, but not when referring to the concept. This is a reference to the concept, hence the lowercase letter.